deployments/.example.local/environment

```diff
@@ -19,6 +19,9 @@ _set AZUL_SUBSCRIBE_TO_DSS 0
 _set AZUL_DOMAIN_NAME "dev.explore.data.humancellatlas.org"
 _set AZUL_SUBDOMAIN_TEMPLATE "{lambda_name}.$AZUL_DEPLOYMENT_STAGE"
 
+_set AZUL_URL_REDIRECT_DOMAIN_NAME "dev.url.data.humancellatlas.org"
```

Comment:
This contradicts your statement from Slack that bucket name and the domain name have to be the same for CNAMEs.

Reply:
I.e. `AZUL_URL_REDIRECT_S3_BUCKET` is used as both the bucket name and the actual redirection url. I know that isn't clearly stated.